README.md

```diff
@@ -137,8 +138,30 @@ Removing a version from the build won't make it be removed from Docker registry,
 
 ## Using and extending
 
+### PHP FPM healthcheck
+
+This image ships with the [php-fpm-healthcheck](https://github.com/renatomefi/php-fpm-healthcheck) which allows you to healthcheck FPM apart from the Nginx setup, providing more compatibility with the single process Docker container.
```

Comment:
```suggestion
This image ships with the [php-fpm-healthcheck](https://github.com/renatomefi/php-fpm-healthcheck) which allows you to healthcheck FPM apart from the Nginx setup, providing more compatibility with a single process Docker container.
```

Reply:
`The` was used intentionally here, it's a reference to https://cloud.google.com/solutions/best-practices-for-building-containers#package_a_single_app_per_container